include/lsst/afw/table/Catalog.h

```diff
@@ -15,6 +15,7 @@
 #include "lsst/afw/table/io/FitsWriter.h"
 #include "lsst/afw/table/io/FitsReader.h"
 #include "lsst/afw/table/SchemaMapper.h"
+#include "lsst/log/Log.h"
```

Comment:
I couldn't find an example where logging is done from another header file. I guess it makes sense in general, but since the implementation of the `find` method is in this header file, and the info about whether or not the catalog was sorted is not propagated up, the logging has to be done from here (if it's done).